### ISSUE TITLE
more fzf commands

### ISSUE THREAD

**Issue by jakeisnt** [jakeisnt/nixcfg#225]
https://github.com/nushell/nushell/issues/5785

i want: 
- nushell integrated wifi selection
- better git integration
- find website by fzfing through browser history (visit ....)
- and more!

**Comment by texastoland** [comment]
IFL these are mostly the realm of community projects. I more constructive wish would be skim integrated as a library somehow. That could provide a compelling UX out of the box in a way that other shells rely on less official integrations.

**Comment by jakeisnt** [comment]
> IFL these are mostly the realm of community projects. A more constructive wish might be skim [integrated as a library](https://github.com/lotabout/skim#use-as-a-library) somehow. That could provide a compelling UX OOTB in an official way whereas other shells rely on less discoverable (often unmaintained) integrations.

I completely agree! Maybe you take the name of the issue literally - it's a note to self to add more "fzf-like" commands to my config. All of the issues on this repo are similar - just ideas. Cool repo though!!!

**Comment by texastoland** [comment]
There have been a few similar issues historically so (as just another user) I like the idea of keeping 1 open for brainstorming 👍🏼

**Comment by jakeisnt** [comment]
Closing and tracking in TODO.md.
